staff/lucas-diaz/prework/calculator-html/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="upperPart">
+            <div class="buttons">
+                <div class="buttonStyle" id="redButton"></div>
+                <div class="buttonStyle" id="yellowButton"></div>
+                <div class="buttonStyle" id="greenButton"></div>
+            </div>
+            <div class="previousResult">
+                <p class="previousResultNumber">
+                    0
+                </p>
+                <p class="previousResultSign">
+                    
+                </p>
+            </div>
+            <div class="results">
+                <p>
+                    0
+                </p>
+            </div>
+        </div>
+        <div class="bottompart">
+            <button class="box resetButton">AC</button>
+            <button class="box squareRootButton">&radic;</button>
+            <button class="box orange divideButton">/</button>
+            <button class="box sevenButton numberButton">7</button>
+            <button class="box eightButton numberButton">8</button>
+            <button class="box nineButton numberButton">9</button>
+            <button class="box orange multiplyButton">*</button>
+            <button class="box fourButton numberButton">4</button>
+            <button class="box fiveButton numberButton">5</button>
+            <button class="box sixButton numberButton">6</button>
+            <button class="box orange substractButton">-</button>
+            <button class="box oneButton numberButton">1</button>
+            <button class="box twoButton numberButton">2</button>
+            <button class="box threeButton numberButton">3</button>
+            <button class="box orange addButton">+</button>
+            <button class="box zeroButton numberButton">0</button>
+            <button class="box dotButton">.</button>
+            <button class="box orange equalButton">=</button>
+        </div>
+    </div>
+    <script src="/main.js"></script>
```

Comment:
cannot load main.js. You need to delete the slash in src: "src='main.js'"

Reply:
changed, thanks